src/environments/environment.ts

```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  url: 'http://voterreg.tbd.com/',
+  url: 'http://us-avx:3000/us/app',
```

Comment:
When I manually toggle production to true and run this locally, I get the following in my browser console when clicking next on the request access code page:

```
GET http://us-avx:3000/us/app/config net::ERR_NAME_NOT_RESOLVED
```

That's not surprising, since I'm guessing this is something you have running locally which I obviously won't. I remember awhile ago we talked about getting a staging site for us non-AV folks to use; will that eventually replace this?

Totally non-blocking, so I'll go ahead and merge - we can follow up either on this thread or elsewhere.

Reply:
Yes, we are still working on getting that running so everyone can try out the entire flow. I'll keep you posted!